### PR TITLE
[IT-1101] add template to setup aws budgets

### PR DESCRIPTION
**Pull request by zaro0508**
This cfng template will allow us to use org-formation to configure
a budget for each AWS account and get notifications when budgets
are exceeding expectations.

### CODE REVIEW

File: templates/Billing/budgets.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            Action: sns:Publish
+            Resource: !Ref NotificationTopic
+
+  # note that changing some attributes on Budget resources will result in an error.
+  # this unfortunately is a cloudformation error described here: https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/365
+  # workaround is to delete the budget resource and then re-add it.
+  Budget:
+    Type: AWS::Budgets::Budget
+    Properties:
+      Budget:
```
Comment (ahayden) [inline templates/Billing/budgets.yaml:57]: Does the budget apply to all resources in the account by default?

**Review by xschildw — APPROVED** [review]
Sounds good to me, wondering if we need 80% actual (if forecasting is any good).

**Review by ahayden — APPROVED** [review]
Looks good

File: templates/Billing/budgets.yaml (hunk 1 above)
Comment (zaro0508) [reply templates/Billing/budgets.yaml:57]: it applies to the entire account so yes.

**Pull request merged**